### PR TITLE
Add user endpoints

### PR DESCRIPTION
**Pull request by Yevhenii-Bezeha**

### CODE REVIEW

File: controllers/usersControllers.js [hunk 1]
```diff
@@ -16,6 +19,166 @@ const signup = async (req, res) => {
...
+      name,
+      avatar,
+    },
+  });
+};
+
+const getUserInfo = async (req, res) => {
+  const userId = req.params.userId;
+
+  const user = await usersServices.findUser({ _id: req.params.userId });
```
Comment (fff500) [inline controllers/usersControllers.js]: Тут можна заюзати змінну `userId`?

Comment (Yevhenii-Bezeha) [reply controllers/usersControllers.js]: Thank you! Fixed )

File: controllers/usersControllers.js [hunk 2]
```diff
@@ -16,6 +19,166 @@ const signup = async (req, res) => {
...
+    throw httpError(401, "Email or password is wrong");
+  }
+
+  const token = await usersServices.createToken({ id: user.id });
+  const updatedUser = await usersServices.updateUser(
+    { _id: user.id },
+    { token }
+  );
+
+  res.status(201).json({
```
Comment (OleksandrBalashov) [inline controllers/usersControllers.js]: Please use 200 status here. And fix the same places 👇 as well

File: controllers/usersControllers.js [hunk 3]
```diff
@@ -16,6 +19,166 @@ const signup = async (req, res) => {
...
+    ...additionalInfo,
+  });
+};
+
+const updateAvatar = async (req, res) => {
+  const { _id } = req.user;
+  const oldPath = req.file.path;
+  const newPath = path.resolve("public/avatars", req.file.originalname);
+
+  await jimp
```
Comment (OleksandrBalashov) [inline controllers/usersControllers.js]: What do you think to move this logic in helpers?

File: controllers/usersControllers.js (hunk 2 above)
Comment (Yevhenii-Bezeha) [reply controllers/usersControllers.js]: Thanks, fixed )

File: controllers/usersControllers.js (hunk 3 above)
Comment (Yevhenii-Bezeha) [reply controllers/usersControllers.js]: Thanks, fixed too)

**Review by OleksandrBalashov — APPROVED** [review]

**Pull request merged**